rally-domain/src/main/groovy/yakworks/rally/orgs/repo/ContactRepo.groovy

```diff
@@ -147,7 +148,11 @@ class ContactRepo extends LongIdGormRepo<Contact> {
     @Override
     void doAfterPersistWithData(Contact contact, PersistArgs args) {
         Map data = args.data
-
+        if(data.getBoolean(IS_PRIMARY)) {
```

Comment:
how do we get this into the rest docs? seems like overkill but maybe we add a transient isPrimary to the domain and only show it on crete update by setting the  `oapi: "C,U"` on the constraint